README.rst

```diff
@@ -0,0 +1,21 @@
+*******
+Martian
+*******
+
+.. image:: https://travis-ci.org/zopefoundation/martian.svg?branch=master
+        :target: https://travis-ci.org/zopefoundation/martian
+
+.. image:: https://readthedocs.org/projects/martian/badge/?version=latest
+        :target: http://martian.readthedocs.org/en/latest/
```

Comment:
Would mildly prefer https://

Note that the link doesn't actually work (neither http:// nor the https:// versions) -- does Martian have any docs uploaded to RTD?

Reply:
(Yes, I didn't notice #6 before reviewing.)